app/src/main/java/com/example/appmarvel/services/usecases/GetCharactersListUseCase.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.appmarvel.services.usecases
+
+import com.example.appmarvel.entity.Character
+import com.example.appmarvel.services.utils.Result
+import org.koin.core.component.KoinComponent
+
+interface GetCharactersListUseCase : KoinComponent {
```

Comment:
the use case interface and its implementation can be in only one file and should be outside the package services

Reply:
Done